tests/python/test_bitpacked_fields.py

```diff
@@ -213,3 +213,16 @@ def verify_val():
 
     set_val()
     verify_val()
+
+
+@test_utils.test()
+def test_invalid_place():
+    with pytest.raises(
+            ti.TaichiCompilationError,
+            match=
+            'At least 2 fields need to be placed when shared_exponent=True'):
```

Comment:
nit: it'll be nicer to palce the context manager around the line that really throws the error so that people who look at the test cases can know immediately where it's expected to throw ;)

Reply:
done